src/plugins/yarn.js

```diff
@@ -22,6 +22,33 @@ export default class Yarn {
   get lockfile (): string { return path.join(this.config.cacheDir, 'yarn.lock') }
   get bin (): string { return path.join(__dirname, '..', '..', 'yarn', `yarn-${this.version}.js`) }
 
+  // https://github.com/yarnpkg/yarn/blob/master/src/constants.js#L73-L90
+  pathKey (): string {
+    let pathKey = 'PATH'
+
+    // windows calls its path "Path" usually, but this is not guaranteed.
+    if (process.platform === 'win32') {
```

Comment:
@dickeyxxx this is hijacked from `yarn` and I am planning on filing a PR for adding a `--scripts-prepend-node-path=true` flag, so I would like to have this as close to their implementation as possible so I would like to leave this as is to keep the PR I submit to them smaller in scope